src/hooks/core/chat/useFetchPostNotificationHook.ts

```diff
@@ -36,6 +48,9 @@ const useFetchPostNotificationHook = () => {
       const anonymousPostNotifications =
         await AnonymousMessageRepo.getAllAnonymousPostNotifications();
       saveNotifications(anonymousPostNotifications, ChannelList.fromAnonymousPostNotificationAPI);
+      // update timestamp
+      const timestamp = new Date().toISOString().split('T')[0];
+      StorageUtils.anonymousNotificationTimeStamp.set(timestamp);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You are storing the current date as the timestamp for anonymous notifications. However, this might not be accurate if the server and client clocks are not synchronized. Consider using the server-provided timestamp if available. If the server doesn't provide one, then this approach is fine as a fallback.

Also, error handling seems to be missing for the `StorageUtils.anonymousNotificationTimeStamp.set(timestamp);` operation. It would be good to wrap this in a try-catch block to handle any potential errors.

```diff
- const timestamp = new Date().toISOString().split('T')[0];
- StorageUtils.anonymousNotificationTimeStamp.set(timestamp);
+ try {
+   const timestamp = new Date().toISOString().split('T')[0];
+   await StorageUtils.anonymousNotificationTimeStamp.set(timestamp);
+ } catch (e) {
+   console.log('error on setting anonymousNotificationTimeStamp:', e);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->